independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/Annotations.java

```diff
@@ -173,4 +187,69 @@ public static AnnotationInstance getParameterAnnotation(MethodInfo method, DotNa
         return null;
     }
 
+    public static Collection<AnnotationInstance> onlyRuntimeVisible(Collection<AnnotationInstance> annotations) {
+        return new OnlyRuntimeVisibleAnnotations(annotations);
+    }
+
+    private static class OnlyRuntimeVisibleAnnotations extends AbstractCollection<AnnotationInstance> {
```

Comment:
I see no point in existence of this class.

Reply:
It's gone now.